src/cluster.c

```diff
@@ -1410,8 +1380,9 @@ static int valkeyClusterSetOptionAddNodes(valkeyClusterContext *cc,
         return VALKEY_ERR;
     }
 
-    address = sdssplitlen(addrs, strlen(addrs), CLUSTER_ADDRESS_SEPARATOR,
-                          strlen(CLUSTER_ADDRESS_SEPARATOR), &address_count);
+    /* Split into individual addresses. */
+    const char *sep = ",";
+    address = sdssplitlen(addrs, strlen(addrs), sep, strlen(sep), &address_count);
```

Comment:
(Minor) The variable is not necessary, I suppose.

```suggestion
    address = sdssplitlen(addrs, strlen(addrs), ",", strlen(","), &address_count);
```